src/libs/tools/s3bench.py

```diff
@@ -190,7 +196,12 @@ def execute_s3bench_workload(self) -> Tuple[bool, Any]:
             if self.skip_cleanup:
                 cmd += "-skipCleanup "
             if self.validate:
-                cmd += "-validate"
+                cmd += "-validate "
+            if self.s3max_retries:
+                cmd = cmd + f"-s3MaxRetries={self.s3max_retries} "
+            if self.connection_timeout:
```

Comment:
better to mention it as httpClientTimeout  as there are many type of timeouts supported.